src/components/Board/Output/SymbolOutput/BackspaceButton/BackspaceButton.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import IconButton from '@material-ui/core/IconButton';
+import BackspaceIcon from '@material-ui/icons/Backspace';
+
+const styles = {
```

Comment:
Can we have a CSS file for these styles?

Reply:
I wish.. last time I checked `material-ui` didn't allow to run them over without `!important`.
Let me try again..

Reply:
Yeh still the same.. so in this case I think placing it inside the JS is okay.
Maybe in the future we'll switch to css in js or something similar to avoid two ways of dealing with css..